index.html

```diff
@@ -0,0 +1,30 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Product Fetching</title>
+    <link rel="stylesheet" href="./style.css"></link>
+  </head>
+  <body>
+    <div class="container">
+      <div class="heading">
+        <div class="logo">Logo</div>
+        <ul class="navs">
+          <li>Home</li>
+          <li>Products</li>
+          <li>About</li>
+          <li>Contact</li>
+        </ul>
+        <div id="btn-group">

```

Comment:
naming convention should be correct. BEM

Reply:
done